lib/oracledb.js

```diff
@@ -687,7 +690,7 @@ function initOracleClient(arg1) {
     errors.throwErr(errors.ERR_THIN_CONNECTION_ALREADY_CREATED);
   }
   if (_initOracleClientArgs === undefined) {
-    const oracledbCLib = _initCLib();
+    const oracledbCLib = _initCLib(arg1);
```

Comment:
This needs to be `options`, not `arg1`!

Reply:
Changes has been applied. Thanks for remind